tuxemon/states/choice/choice_item.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from collections.abc import Callable, Sequence
+from typing import Any
+
+import pygame_menu
+from pygame_menu.locals import POSITION_CENTER, POSITION_EAST
+from pygame_menu.widgets.selection.highlight import HighlightSelection
+
+from tuxemon import prepare
+from tuxemon.db import db
+from tuxemon.menu.menu import PygameMenuState
+from tuxemon.menu.theme import get_theme
+from tuxemon.tools import transform_resource_filename
+
+ChoiceMenuGameObj = Callable[[], None]
+
+
+def fix_width(screen_x: int, pos_x: float) -> int:
```

Comment:
Could we add more documentation for the parameters of this method?

I'm not sure to understand what's supposed to be `screen_x` for example.

Reply:
I added fix_width and fix_height months ago (when I coded the Journal), by looking at both now these are identical. So I did something bigger: renamed one in fix_measure, clarified and replaced.